src/main/java/io/cryostat/jfr/datasource/server/JfrResource.java

```diff
@@ -15,31 +15,25 @@
 import io.cryostat.jfr.datasource.sys.FileSystemService;
 
 import io.quarkus.vertx.web.Route;
+import io.quarkus.vertx.web.Route.HttpMethod;
 import io.smallrye.common.annotation.Blocking;
-import io.vertx.core.http.HttpMethod;
 import io.vertx.core.http.HttpServerResponse;
 import io.vertx.core.json.JsonObject;
-import io.vertx.core.logging.Logger;
-import io.vertx.core.logging.LoggerFactory;
 import io.vertx.ext.web.FileUpload;
 import io.vertx.ext.web.RoutingContext;
 import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class JfrResource {
-    private static final Logger LOGGER = LoggerFactory.getLogger(RecordingService.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(JfrResource.class);
```

Comment:
Previously used with `RecordingService.class` in `JftResource`. I am not sure if it is intentional or a typo.

Reply:
Seems like it was most likely just a typo or copy-paste error.